bsconfig.json

```diff
@@ -13,5 +13,8 @@
   "bs-dev-dependencies": [
     "@glennsl/bs-json"
   ],
-  "refmt": 3
-}
\ No newline at end of file
+  "refmt": 3,
+  "warnings": {
+    "number": "-105"
```

Comment:
Unrelated to the cookie update; ignore new bucklescript warning "BuckleScript warning: redirect : the external name is inferred from val name is unsafe from refactoring when changing value name"